### PR TITLE
Add node-sass filter support

### PR DESCRIPTION
**Pull request by gremo**
Added the node-sass filter and tests.

### CODE REVIEW

**Comment by gremo** [comment]
Any news on this? Thanks.

**Comment by kminek** [comment]
any update on this? bootstrap 4 is around the corner, it's using sass by default... and there are people who would like to avoid installing ruby... :D

**Comment by soullivaneuh** [comment]
This MR is a must have, please review! :D

BTW @gremo, travis is failing. Could you please check?

**Comment by gremo** [comment]
@Soullivaneuh It seems something related composer not downloading form cssmin url:

```
[Composer\Downloader\TransportException]                                     
Your configuration does not allow connections to http://cssmin.googlecode.c  
om/files/cssmin-v3.0.1.php. See https://getcomposer.org/doc/06-config.md#se  
cure-http for details.  
```

Something I can't change. If you need to refactor or change the SASS filter code, just ask!

BTW; I'm going to update the fork and rebase, hope this can help.

**Comment by aik099** [comment]
@gremo , you should use `natxet/cssmin` package (as Assetic is doing for tests), which is a clone from Google Code.

**Comment by gremo** [comment]
@aik099 @Soullivaneuh after rebasing the cssmin filter error is gone. Any help solving the remaining would be appreciated!

**Comment by soullivaneuh** [comment]
@gremo Seems to be unrelated failing test, I'll try to take a look.

**Comment by soullivaneuh** [comment]
@gremo please see https://github.com/kriswallsmith/assetic/pull/846

File: src/Assetic/Filter/NodeSassFilter.php [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+use Assetic\Asset\AssetInterface;
+use Assetic\Exception\FilterException;
+
+/**
+ * Node-sass filter.
+ *
+ * @link https://github.com/sass/node-sass
+ * @author Marco Polichetti <gremo1982@gmail.com>
+ */
+class NodeSassFilter extends BaseProcessFilter
```
Comment (soullivaneuh) [inline src/Assetic/Filter/NodeSassFilter.php:23]: Why not extending `BaseSassFilter`?

**Comment by soullivaneuh** [comment]
Just tested your filter @gremo, works great! :+1:

**Comment by pensiero** [comment]
+1 it would be very useful this filter

**Comment by gremo** [comment]
Is there something I can do? Ping @Soullivaneuh

**Comment by gremo** [comment]
And... still waiting, after 10 months. Can we please check what needs to be done in order to get this merged, thanks?

**Comment by soullivaneuh** [comment]
> Is there something I can do? Ping @Soullivaneuh

Unfortunately I'm not a maintainer of this project. cc @kriswallsmith

**Comment by gremo** [comment]
I ended up creating a new repository https://github.com/gremo/assetic-extra please contribute! Cheers!

**Comment by gremo** [comment]
Closing this PR as It will never be merged. Help me here: https://github.com/gremo/assetic-extra

**Pull request closed without merging**